exercises/robot-simulator/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#[derive(PartialEq, Debug)]
```

Comment:
I was only adding what I needed in order to get the compiler to compile. I imagine the real solution will have more derived traits.